airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4569,6 +4608,7 @@ void loop(void) {
 			data += result_PMS;
       		sum_send_time += sendCFA(result_PMS, PMS_API_PIN, FPSTR(SENSORS_PMSx003), "PMS_");
 			sum_send_time += sendSensorCommunity(result_PMS, PMS_API_PIN, FPSTR(SENSORS_PMSx003), "PMS_");
+			sum_send_time += sensor_readings.println(result_PMS); // Log PMSX003 sensor data to microSD card
```

Comment:
@Warukira would it be better to have a logger function?

Reply:
@chegejames , do you mean have single logger function, instead of the current way we are logging data per sensor?

Reply:
@chegejames , implemented the requested changes. Kindly review.